doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-standalone-editors.adoc

```diff
@@ -0,0 +1,152 @@
+[id="proc-standalone-editors_{context}"]
+= Configuring the {PRODUCT} standalone editors
+
+The {PRODUCT} provides standalone editors that are distributed in a self-contained library providing an all-in-one JavaScript file for each editor. The JavaScript file uses a comprehensive API to set and control the editor.
```

Comment:
Drop "The", just "{PRODUCT} provides ..."

Reply:
Done